### DISCUSSION TITLE
Delayed mind control

### DISCUSSION THREAD

**Discussion by 8abs** [Phobos-developers/Phobos#1018] [Ideas and feedback]
### First check

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search and read the issue list to find a similar issue and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

### Description

Realizing delayed mind control effect by using Ivan logic or other types of logic is still not possible. Basic thought is close to phobos create unit logic: warhead could memorize the owner, so that when it's due time, the target would be controlled using that stored owner information.

### INI code

```ini
[SomeSection]
SomeKey=SomeValue
```

**Comment by 8abs** [comment]
This really could create lots of terror and joy in game. Like "There is a traitor but I don't know who he is, and I can't kill all of my units only to find them."

**Comment by mevitar** [comment]
There are already ways to delay the mind control effect without Ivan Bombs: a slow projectile, PreImpactAnim, or a superweapon. If someone really wants to, they can simulate a traitor among enemy ranks with these (a superweapon that affects random units without any input on either side would be probably best suited for that).
